tests/includes/output.sh

```diff
@@ -0,0 +1,26 @@
+OUTPUT() {
+    local output
+
+    output=${1}
+    shift
+
+    if [ -z "${output}" ] || [ "${VERBOSE}" -gt 1 ]; then
+        echo
+    fi
+
+    while read data; do
+        if [ -z "${output}" ]; then
```

Comment:
It did take me a while to figure out what the various if blocks do here. It would be great if you could add some comments for others examining this part of the code in the future.